
 This patch will prevent the Win32 FD from printing an error message
 when it attempts to restore the permissions for a drive (which Win32
 doesn't permit). The error is harmless in any case.
 Apply the patch to 1.36.2 with

 cd <bacula-source>
 patch -p0 <1.36.2-win32-drive.patch
 make
 ...<F12>


Index: src/findlib/create_file.c
===================================================================
RCS file: /cvsroot/bacula/bacula/src/findlib/create_file.c,v
retrieving revision 1.34
diff -u -r1.34 create_file.c
--- src/findlib/create_file.c	27 Jul 2004 21:00:33 -0000	1.34
+++ src/findlib/create_file.c	9 Mar 2005 17:52:50 -0000
@@ -271,6 +271,12 @@
 	 if ((bopen(bfd, attr->ofname, O_WRONLY|O_BINARY, 0)) < 0) {
 	    berrno be;
 	    be.set_errno(bfd->berrno);
+#ifdef HAVE_WIN32
+	    /* Check for trying to create a drive, if so, skip */
+            if (attr->ofname[1] == ':' && attr->ofname[2] == '/' && attr->ofname[3] == 0) {
+	       return CF_SKIP;
+	    }
+#endif
             Jmsg2(jcr, M_ERROR, 0, _("Could not open %s: ERR=%s\n"), 
 		  attr->ofname, be.strerror());
 	    return CF_ERROR;
